databases/week1/database-week1.sql

```diff
@@ -0,0 +1,89 @@
+
+SELECT `status`.`id`,
+    `status`.`name`
+FROM `homewrk DB`.`status`;
+
+
+SELECT `task`.`id`,
+    `task`.`title`,
+    `task`.`description`,
+    `task`.`created`,
+    `task`.`updated`,
+    `task`.`due_date`,
+    `task`.`status_id`,
+    `task`.`user_id`
+FROM `homewrk DB`.`task`;
+
+
+SELECT COUNT(*)
+FROM `homewrk DB`.`task`;
+
+/* 1.Find out how many tasks are in the task table */
+SELECT `task`.`due_date`
+FROM `homewrk DB`.`task`
+WHERE `task`.`due_date` is NULL;
+--35
+
+/*2.Find out how many tasks in the task table do not have a valid due date*/
+SELECT *
+FROM `status` , `task`
+WHERE `status`.`name` = 'Done' 
+AND `status`.`id` = `task`.`status_id`;
+--8
+
+/*3.Find all the tasks that are marked as done*/
+SELECT *
+FROM `status` , `task`
+WHERE `status`.`name` != 'Done' 
+AND `status`.`id` = `task`.`status_id`;
+
+/*4.Find all the tasks that are not marked as done*/
+SELECT *
+FROM `task`
+ORDER BY `task`.`created` DESC;
+
+/*5.Get all the tasks, sorted with the most recently created first*/
+SELECT `user`.`id`,
+    `user`.`name`,
+    `user`.`email`,
+    `user`.`phone`
+FROM `homewrk DB`.`user`;
```

Comment:
This query is bit confusing for me, it is related to 'task' table or 'user' table.

Reply:
Sorry i will check and solve